src/tests/ftest/server/daos_server_dump.py

```diff
@@ -25,43 +26,103 @@ def __init__(self, *args, **kwargs):
         self.start_servers_once = False
         self.setup_start_agents = False
 
+    def tearDown(self):
+        # pylint: disable=unused-private-member
+        """Tear down after each test case."""
+        super().tearDown()
+
+        # force test status !!
+        self.__status = 'PASS'
+
+        # DAOS-1452 may need to check for one file per engine...
+        ret_codes = pcmd(self.hostlist_servers, r"ls /tmp/daos_dump*.txt")
+        # Report any failures
+        if len(ret_codes) > 1 or 0 not in ret_codes:
```

Comment:
From what I understand, it should be better to test if there is at least one key different from 0.
With your actual codes, the test should pass if ret_codes contains only value different from 0.
If my understanding is correct, one solution could be to create a new map:
`
ret_codes = {key:value for(key ,value) in ret_codes.items() if key != 0}
if len(ret_codes) > 1:
   ....
`

Reply:
hmm, do you mean that current test does not cover all error return cases ?

Reply:
> hmm, do you mean that current test does not cover all error return cases ?

From what I understand, it seems to be the case.

Reply:
Forgot what I said